src/encoded/audit/experiment.py

```diff
@@ -1077,6 +1085,49 @@ def check_experiment_wgbs_encode3_standards(experiment,
     return
 
 
+def check_experiment_wgbs_encode4_standards(
+    experiment,
+    files_structure,
+    organism_name,
+    desired_assembly
+):
+    alignment_files = files_structure.get('alignments').values()
+    fastq_files = files_structure.get('fastq_files').values()
+    cpg_quantifications = files_structure.get('cpg_quantifications').values()
+
+    if fastq_files == []:
+        return
+
+    yield from check_wgbs_read_lengths(fastq_files, organism_name, 130, 100)
+
+    read_lengths = get_read_lengths_wgbs(fastq_files)
+
+    pipeline_title = scanFilesForPipelineTitle_not_chipseq(
+        alignment_files, ['GRCh38', 'mm10'], ['gemBS'])
+
+    if pipeline_title is False:
+        return
+
+    if 'replication_type' not in experiment or experiment['replication_type'] == 'unreplicated':
+        return
+
+    gembs_metrics = get_metrics(
+        alignment_files, 'GembsAlignmentQualityMetric', desired_assembly)
+    cpg_metrics = get_metrics(
+        cpg_quantifications, 'CpgCorrelationQualityMetric', desired_assembly)
+    samtools_metrics = get_metrics(
+        cpg_quantifications, 'SamtoolsFlagstatsQualityMetric', desired_assembly)
+
+    yield from check_wgbs_coverage_ENCODE4(
+        gembs_metrics,
+        pipeline_title,
+        get_pipeline_objects(alignment_files))
+    yield from check_wgbs_pearson_ENCODE4(cpg_metrics, 0.8, pipeline_title)
+    yield from check_wgbs_lambda_ENCODE4(gembs_metrics, 1, pipeline_title)
```

Comment:
Conversion rate is expressed as a float between 0 and 1. We want to audit values less than 0.99/99%:
```suggestion
    yield from check_wgbs_lambda_ENCODE4(gembs_metrics, 0.99, pipeline_title)
```